app/src/main/java/protect/card_locker/AboutActivity.java

```diff
@@ -62,6 +62,7 @@ protected void onCreate(Bundle savedInstanceState) {
         }
 
         final List<ThirdPartyInfo> USED_LIBRARIES = new ArrayList<>();
+        USED_LIBRARIES.add(new ThirdPartyInfo("Yalantis(Image Cropping)","com.github.yalantis:ucrop","Apache 2.0"));
```

Comment:
Yalantis is the developer, not the library name. It's called uCrop

com.github.yalantis:ucrop is not a valid URL, the library is at https://github.com/Yalantis/uCrop.

Reply:
file updated